src/components/TimeCardPage/SubmitCard.tsx

```diff
@@ -8,39 +8,57 @@ import Button from 'react-bootstrap/Button';
 
 export default function SubmitCard(props) {
 
-    const [submitted, setSubmitted] = useState(false); 
-    const [submitDate, setSubmitDate] = useState(null); 
+    const CardState = {
```

Comment:
Now that we have typescript, use `enum` for this and extract it into a `types.ts` file (can be in this same directory) so that this type can be commonly reused